m4opt/missions/_ultrasat.py

```diff
@@ -0,0 +1,73 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from regions import RectangleSkyRegion
+from synphot import Gaussian1D, SpectralElement
+
+from .. import skygrid
+from ..constraints import (
+    EarthLimbConstraint,
+    MoonSeparationConstraint,
+    SunSeparationConstraint,
+)
+from ..models import Detector
+from ..models.background import GalacticBackground, ZodiacalBackground
+from ..orbit import TLE
+from ..utils.dynamics import Slew
+from ._core import Mission
+
+ultrasat = Mission(
+    name="ultrasat",
+    fov=RectangleSkyRegion(
+        center=SkyCoord(0 * u.deg, 0 * u.deg), width=14.28 * u.deg, height=14.28 * u.deg
+    ),
+    constraints=[
+        EarthLimbConstraint(48 * u.deg),
+        SunSeparationConstraint(70 * u.deg),
+        MoonSeparationConstraint(35 * u.deg),
+    ],
+    detector=Detector(
+        npix=4 * np.pi,
+        # Pixel scale, 5.4 arcsec / pixel
+        plate_scale=(5.4 * u.arcsec) ** 2,
+        # Circular aperture with a diameter of 33 cm
+        area=np.pi * np.square(0.5 * 33 * u.cm),
+        bandpasses={
+            "NUV": SpectralElement(
+                Gaussian1D,
+                amplitude=0.25,
+                mean=2600 * u.angstrom,
+                stddev=340 * u.angstrom,
+            ),
+        },
+        # FIXME: Add models for Zodiacal light, Cerenkov radiation, and Stray light
```

Comment:
There is already a zodiacal light term below.

```suggestion
        # FIXME: Add models for Cerenkov radiation and stray light
```